src/ngraph/frontend/onnx_import/CMakeLists.txt

```diff
@@ -31,7 +31,18 @@ add_library(onnx_import_interface OBJECT
     onnx.proto)
 
 add_library(onnx_import STATIC
-    onnx.pb.cc)
+    onnx.pb.cc
+    attribute.cpp
+    attribute.hpp
```

Comment:
Done in #1213

Reply:
Noooo... Please keep it as is. CMake does REQUIRE all files comprising target on the list in order to prepare dependencies correctly. Furthermore removing them from the list will cause various tools (i.e. Clion, the tool I am using for development) will stop work correctly (some symbols will not be recognized because of that). In pre-modern CMake era developer used e.g. 'file(GLOB SRCS *.hpp)' for the very same reason. Please keep it as is!